connectors/slack/src/main/java/io/camunda/connector/slack/inbound/model/SlackWebhookProperties.java

```diff
@@ -8,14 +8,39 @@
 
 import com.slack.api.app_backend.SlackSignature;
 import io.camunda.connector.api.inbound.webhook.VerifiableWebhook;
+import io.camunda.connector.generator.dsl.Property;
+import io.camunda.connector.generator.java.annotation.TemplateProperty;
+import io.camunda.connector.generator.java.annotation.TemplateProperty.PropertyType;
+import jakarta.validation.constraints.NotBlank;
 import java.util.Map;
 import java.util.function.Function;
 
 public record SlackWebhookProperties(
-    String context,
-    String slackSigningSecret,
-    Function<Map<String, Object>, VerifiableWebhook.WebhookHttpVerificationResult>
-        verificationExpression) {
+    @TemplateProperty(
+            id = "context",
+            label = "Webhook ID",
+            group = "endpoint",
+            description = "The webhook ID is a part of the URL endpoint")
```

Comment:
I would also add `feel = FeelMode.disabled` to this property and `slackSigningSecret`. We can change the default behavior for inbound connectors to set `disabled` (As we discussed last week with @Oleksiivanov)

Reply:
I created this issue, let's rather change the default value: https://github.com/camunda/connectors/issues/2282

I will tackle it later this week (currently focused on a different task), or feel free to take the lead if you feel like it :)

Reply:
Good point, I'll change that to `disabled`

Reply:
@chillleader it seems like `disabled` is not working for inbound connectors. Setting `feel = Property.FeelMode.disabled` had no effect. Except the field is `hidden`, then it works. 🤔

Reply:
> Setting feel = Property.FeelMode.disabled had no effect.

Good catch, I could confirm the issue - here is the proposed fix: https://github.com/camunda/connectors/pull/2286